roles/docker-garbage-collection/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# file path and name variables
+garbage_home: "/apis/docker_garbage_collection/"
+backup_images: "{{garbage_home}}backup_images/"
```

Comment:
Could you rename this variable be like `backup_images_path` or something like that? The current name is a bit confusing and sounds more like docker images will be used.